src/test/java/org/assertj/vavr/api/MultimapAssert_isNotEmpty_Test.java

```diff
@@ -0,0 +1,49 @@
+package org.assertj.vavr.api;
+
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ * <p>
+ * Copyright 2012-2019 the original author or authors.
+ */
+
+import io.vavr.collection.HashMultimap;
+import io.vavr.collection.Multimap;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.error.ShouldNotBeEmpty.shouldNotBeEmpty;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.vavr.api.VavrAssertions.assertThat;
+
+class MultimapAssert_isNotEmpty_Test {
+
+    @Test
+    void should_pass_if_Multimap_is_not_empty() {
+        assertThat(HashMultimap.withSeq().of("key", "value")).isNotEmpty();
+    }
+
+    @Test
+    void should_fail_when_Multimap_is_null() {
+        assertThatThrownBy(
+                () -> assertThat((Multimap<String, String>) null).isNotEmpty()
+        )
+                .isInstanceOf(AssertionError.class)
+                .hasMessage(shouldNotBeNull().create());
+    }
+
+    @Test
+    void should_fail_if_Multimap_is_empty() {
```

Comment:
Shouldn't test methods be `public`?

Reply:
Not really, since we're using JUnit 5 and it does not require public classes and methods